go/vt/vtgate/resolver_test.go

```diff
@@ -15,12 +15,11 @@ import (
 
 	mproto "github.com/youtube/vitess/go/mysql/proto"
 	"github.com/youtube/vitess/go/vt/key"
+	pb "github.com/youtube/vitess/go/vt/proto/topodata"
```

Comment:
Why this change? I mean, it doesn't matter, just curious. In all files, we define the protobuf import separately, for clarity, as they;re not libraries, but auto-generated, so it feels a bit cleaner.

(I'm not attached to this, I just find it a bit clearer, if others disagree, I'm fine too).